pkg/reconciler/pipelinerun/controller.go

```diff
@@ -13,14 +13,13 @@ func SetupNewReconcilerWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&pipelinev1beta1.PipelineRun{}, builder.WithPredicates(predicate.Funcs{
 			CreateFunc: func(e event.CreateEvent) bool {
-				return true
+				return false
 			},
 			UpdateFunc: func(e event.UpdateEvent) bool {
 				return true
 			},
 			DeleteFunc: func(e event.DeleteEvent) bool {
-				//TODO possibly change to false over time
-				return true
+				return false
```

Comment:
this is fine / not surprising, but it does remind me, I put a quick blurb in the jira, we need at some point to start thinking about our pruning strategy.  Or has thought been put into it and I missed it?

the owner ref usage will help with pruning ... i.e. deleting a DB or ABR will delete any tekton objects they create/own

Reply:
Once the component is built we can technically prune everything, but we will likely want to stash the results somewhere in case people want info from the logs. We probably want to stash these in the same long term storage as the artifacts (e.g. S3 ATM).